kmq.js

```diff
@@ -99,6 +103,20 @@ const parseMessage = (message, botPrefix) => {
     }
 }
 
+const hookTopGGCount = () => {
+    if (!config.topGGToken) {
+        logger.info("No top.gg token passed (check your config.json)! Ignoring posting top.gg server count.");
+        return;
+    }
+    const dbl = new DBL(config.topGGToken, client);
```

Comment:
This will go out of scope and get garbage collected
https://stackoverflow.com/a/38449557

Reply:
I removed the function. I also refactored this part a bit so that it shouldn't be garbage collected anymore.